### ISSUE TITLE
"SyntaxError: Unterminated string in JSON" for large pageContext JSON and slow connection

### ISSUE THREAD

**Issue by brillout** [brillout/vite-plugin-ssr#567]
### Error Message + Error Stack

```shell
SyntaxError: Unterminated string in JSON at position 11161
    at JSON.parse (<anonymous>)
    at te (chunk-00542f98.js:formatted:861:20)
    at dt (chunk-00542f98.js:formatted:881:15)
    at Mt (chunk-c35d5d71.js:formatted:994:15)
```

**Comment by brillout** [comment]
VPS injects the pageContext JSON data *after* `<script>`:

```
<script type="module" src="/assets/entry-client-routing.89eb5a1f.js" defer></script>
<link rel="modulepreload" href="/assets/pages/index.page.9e7181a6.js" as="script" type="text/javascript">
<link rel="modulepreload" href="/assets/chunk-69c0edb3.js" as="script" type="text/javascript">
<link rel="modulepreload" href="/assets/chunk-585e3384.js" as="script" type="text/javascript">
<link rel="modulepreload" href="/assets/chunk-dae4001f.js" as="script" type="text/javascript">
<link rel="modulepreload" href="/assets/chunk-c9dc3e1a.js" as="script" type="text/javascript">
<link rel="modulepreload" href="/assets/renderer/_default.page.client.50b93bc8.js" as="script" type="text/javascript">
<link rel="modulepreload" href="/assets/chunk-56e5657a.js" as="script" type="text/javascript">
<script id="vite-plugin-ssr_pageContext" type="application/json">{"pageContext":{"_pageId":"/pages/index","pageProps":"!undefined","documentProps":"!undefined","someAsyncProps":42}}</script>
```

It's not a problem because the spec says that `<script type="module"/>` shouldn't interupt HTML parsing. But some browser may do some funky things here.

Let's see if https://github.com/brillout/vite-plugin-ssr/commit/13836ebcefe70838a037c93602c40428f37f6cac fixes it.

Closing but let me know if you still run into the problem and I'll re-open this ticket.

**Comment by brillout** [comment]
Fix released in `0.4.63`.
